### ISSUE TITLE
older logs won't convert

### ISSUE THREAD

**Issue by joshbis** [pj4533/PokerNowConvert#7]
Tried to convert some logs from July 4 & 11 and just get a blank box. I wonder if there was some minor change in the formats?

**Comment by pj4533** [comment]
@joshbis    Yeah....there have been several changes.    I thoooooought I added feedback for this case, but might not have made it to the website output (and only shows on the command line tool output).    If you want to attach the log here, I'll verify it and add code to show a good msg (and not just a blank box).  

Unfortunately, I doubt those will work anymore.    Depending on the exact date of the logs, you might convert them with an older version of my command line tool.

**Comment by joshbis** [comment]
Thanks -- no error in the heroku interface. I'll try to grab an older version of pn2ps.

**Comment by pj4533** [comment]
@joshbis    ok I tracked this down for you.   the change was the one to support seat numbers.   that was a breaking change in my converter logic so I set the unix timestamp of supported logs to 1594731595  (7/14/2020).   pn2ps will output a nice error message for these logs but I just haven't change the website to output that message.  I'll make an issue to do that and reference this issue.

If you grab a version of pn2ps prior to 7/14, it should work converting those logs.  If you have trouble, just let me know.
